### ISSUE TITLE
[Lore Question/Suggestion] Resonance of the Faceted Crystal

### ISSUE THREAD

**Issue by keraldi** [WayofTime/BloodMagic#1330]
### Foreword

Resonance of the Faceted Crystal splits Raw Demonic Will into its specialized aspects.
However, it assigns them to elemental inscribed Ritual Stones. This makes one (myself and the guy I play with) believe that the Demonic Will can be categorized and described with elemental traits. 

TL;DR (The question is at the very bottom)

---
However, I would understand an original interpretation (without the assignment to Ritual Stones that happens through the Resonance of the Faceted Crystal) as Water (cannot be harmed, flows back to its original state), Acid (Corrosion, often imagined as green liquid), Anger/Wrath (Vengeance, wrath is often pictured in red) and (very loose,) Explosion, kinetic energy (though winds) (Destruction, Air pictured in yellow).

### Elements in philosophy/culture

Now, let us take a look at the [japanese elements](https://en.wikipedia.org/wiki/Classical_element#Japan) which have been inspiration for many interpretations of the four (or 5 if you include void) elements. [The main article for the five japanese elements is also a good read, however didn't fit quite for what I wanted to point at.](https://en.wikipedia.org/wiki/Five_elements_(Japanese_philosophy))

Earth = solid objects, stone, stubbornness, stability
Water = liquid/fluid things, adaptability, flexibility
Fire = destruction, energy, force, heat, desire
Air/Wind = movement, elusiveness, compassion

### Comparison to Demonic Will

We have 4 aspects of Demonic Will: 
- Steadfast (Blue), in combat representing defense, which I personally would assign to the element Earth (as in fortification) (Ritual Stone of Earth is green)
- Corrosive (Green), in combat representing poison, adaptability, (corrosion/acid), or corrupted Water (Ritual Stone of Water is blue)
- Destructive (Yellow-ish), in combat representing power, energy, force, represents the negative/destructive traits of fire (exploding arrows), aka. Fire (Ritual Stone of Fire is red)
- Vengeful (Red), in combat representing movement, speed, the elusive warrior, corrupted comassion, similar to Air (Ritual Stone of Air is yellow)

Bonus: Raw Will is not aspected (or all aspects) and the first Demonic Will we encounter. At that time a mystery to us. It could be assigned to Void/Aether/Sky as otherworldly, unknown. Or these are Dusk and Dawn.

### What I would like to see

Basically recoloring Demonic Will to be more fitting to the classic interpretation of elements and adding lore to them being corrupted (demonic) representations of elemental power. This would mean that Corrosive becomes blue (which, in my opinion, is the only fitting color as Acid is often pictured with green color), Destructive Will becomes red, Vengeful Will becomes yellow and finally, Steadfast Will becomes Green.

## The Question:

Is there any connection between elements and Demonic Will, is there any deeper meaning as to why the Resonance of the Faceted Crystal lets the crystals grow on these Ritual Stones?

**Comment by keraldi** [comment]
Still unanswered. I demand a refund.

**Comment by wrincewind** [comment]
> Is there any connection between elements and Demonic Will, is there any deeper meaning as to why the Resonance of the Faceted Crystal lets the crystals grow on these Ritual Stones?

Yes. 

If this answer does not satisfy, your refund check will be automatically issued. (there may be a small $50 processing fee. We accept Greek obols, thrupenny bits, or Yapese _Rai_ stones.)
